dotnetapp-3.5/Dockerfile

```diff
@@ -15,3 +15,12 @@ WORKDIR /app
 COPY . ./
 
 RUN ["msbuild.exe", "dotnetapp-3.5.csproj", "/p:Configuration=Release"]
+
+
+FROM microsoft/dotnet-framework:3.5
+
+WORKDIR /app
+
```

Comment:
Nit: I don't think all of the white space between every instruction is necessary.  Consider reducing and making it consistent with the 4.7.1 Dockerfile.